VideoLocker/assets/config/registration_form.json

```diff
@@ -35,7 +33,6 @@
         "errorMessages": {},
         "placeholder": "",
         "defaultValue": "",
```

Comment:
Thanks for the reply here @catong. We should either: 
1.) merge this but not update the apps until we resolve the inclusion of these support labels.
2.) or include the text labels within this PR (whether it is a separate ticket or not is a separate issue)